projects/core/src/product/services/product-page-meta.resolver.ts

```diff
@@ -94,11 +94,18 @@ export class ProductPageMetaResolver extends PageMetaResolver
   ): Observable<any[]> {
     const breadcrumbs = [];
     breadcrumbs.push({ label: breadcrumbLabel, link: '/' });
-
-    for (const c of product.categories) {
+    if (product.categories[0]) {
+      const { name, code } = product.categories[0];
+      breadcrumbs.push({
+        label: name || code,
+        link: `/c/${code}`,
+      });
+    }
+    if (product.categories[1]) {
+      const { name, code } = product.categories[1];
       breadcrumbs.push({
-        label: c.name || c.code,
-        link: '/c/' + c.code,
+        label: name || code,
+        link: `/Brands/${name}/c/${code}`,
```

Comment:
Can you use `SemanticPathService.transform` to generate url from config?

Reply:
applied discussed solution